config/qualifications.yml

```diff
@@ -1,21 +1,31 @@
-# A fully-trained programmer should
 ---
-title: 'Have demonstrated understanding of Ruby core methods / Rails object extensions'
+title: 'Configure your laptop for development'
 description: >
+  go to https://strap.umasstransit.it/ and follow the instructions.
 subtasks:
-  - 'Ruby object types'
-  - 'Methods and functionality brought by Rails (try, 1.week, etc.)'
 ---
-title: 'Have demonstrated familiarity with MVC architecture and the Rails asset pipeline'
+title: 'Join UMTS on Github'
 description: >
+  Ask one of your supervisors or team leaders to get you added to
+  the relevant groups in our Github organization.
 subtasks:
-  - 'When to place logic in controller / model'
-  - 'What not to put in a view'
-  - 'Partials'
-  - 'Working securely with params (guarding at the borders)'
 ---
-title: 'Routing, especially member vs. collection routes'
+title: 'Read through documents related to university owned computers'
 description: >
+  Since you will be trusted with university-owned computers and potentially
+  sensitive data, you should review these documents
 subtasks:
-  - 'Default resource CRUD routes'
-  - 'Difference between GET and POST / links and buttons'
+  - '[Security Checklist](https://www.umass.edu/it/support/security/security-checklist-university-owned-computers)'
+  - '[Understanding Sensitive Data](https://www.umass.edu/it/support/security/understand-sensitive-data-umass-amherst)'
+  - '[Data Classification]9https://www.umass.edu/it/support/security/data-classification-umass-amherst)'
+---
+title: 'Read the transportation programmer policies'
+description: >
+  Ask your supervisor to send you this document via email if they have not
```

Comment:
Can we find a place to put this? Do programmers get Box access?

Reply:
There's no reason they can't, and I see no reason they shouldn't--especially because we can have such granular control as to allow access to just one folder.

Reply:
I made a box folder called "Programmer Training Documents" or something. I uploaded the one document. I do not know what the best way to share that link is. I know about sharable links, but it's not that simple--I think you have to invite people to view it. @sherson , what's the best way to go about this part?

Reply:
@sherson !!!!!!!

Reply:
Some options are making a box link public and sharing it, sharing it with individual box users, or sharing it with a group.

Already on my list was to ask UMassIT about creating some new box groups anyway (I don't think we have the ability to do that ourselves); I'll see about creating one for `Transportation Programmer Trainees`.

We'll talk tomorrow.

Reply:
can this maybe not be a blocking concern for getting this out?

Reply:
@Anbranin: We didn't end up connecting about this a couple weeks ago, but FYI there is now a programmer trainee group: `Transportation Programmer Trainees`. I put the trainees in the group, and shared [the folder](https://umass.app.box.com/folder/77237337598) with the group.

(on a side note, we now have several other box groups as well)